decidim-comments/app/cells/decidim/comments/comment/votes.erb

```diff
@@ -1,11 +1,11 @@
 <div class="comment__votes">
   <% if user_signed_in? %>
-    <%= button_to decidim_comments.comment_votes_path(model, weight: 1), remote: true, disabled: voted_down?, class: votes_up_classes, title: t("decidim.components.up_vote_button.text") do %>
+    <%= action_authorized_button_to decidim_comments.comment_votes_path(model, weight: 1), resource: model.commentable, remote: true, disabled: voted_down?, class: votes_up_classes, title: t("decidim.components.up_vote_button.text") do %>
```

Comment:
what about this:
we should put this in a helper in the cell.
If the commentable has a component, add the current_component context, if not just use the normal `button_to`